lib/pcb/convert-circuit-json-to-pcb-svg.ts

```diff
@@ -47,6 +49,10 @@ interface Options {
   height?: number
   shouldDrawErrors?: boolean
   shouldDrawRatsNest?: boolean
+  mode?: {
```

Comment:
You have to also add a type for the regular mode, even if its not used.

i think the structure of this object is a bit confusing. What if we had a separate function convertCircuitJsonToSolderPasteMask(circuitJson, { layer: “top” })?